### PR TITLE
Setup Amazon S3

### PR DESCRIPTION
**Pull request by ibsa-ejeta**
### What this PR does (required):
- sets up amazon s3 for file upload.
- tested it with a function which is commented out
- the main function is to be integrated with other files
- sample image is added to routes folder to generate the following url 
- url https://teamruntimeterror.s3.amazonaws.com/routes/loading1.gif

### Screenshots / Videos (required):
- Post at least one screenshot to show the feature in action

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: server/routes/uploadProfileImage.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }),
+});
+
+app.post("/upload", uploadS3.single("file"), (req, res) => {
+  console.log(req.file);
+});
+
+app.listen(process.env.PORT || 3009);
+
+// const uploadFile = (fileName) => {
```
Comment (rajivtitus) [inline server/routes/uploadProfileImage.js:30]: Please uncomment code here

File: server/routes/uploadProfileImage.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+//     Bucket: "teamruntimeterror",
+//     Key: "routes/loading1.gif",
+//     Body: fs.readFileSync(fileName),
+//   };
+
+//   s3.upload(params, function (err, data) {
+//     if (err) {
+//       throw err;
+//     }
+//     console.log(`File uploaded successfully. ${data.Location}`);
```
Comment (rajivtitus) [inline server/routes/uploadProfileImage.js:41]: Maybe send the file location URL as a response rather than logging it to the console

**Review by rajivtitus — APPROVED** [review]
LGTM! Just a couple of small comments, please have a look. Thanks!

File: server/routes/uploadProfileImage.js (hunk 1 above)
Comment (ibsa-ejeta) [reply server/routes/uploadProfileImage.js]: replaced the whole file with a new one which works on both conditions

File: server/routes/uploadProfileImage.js (hunk 2 above)
Comment (ibsa-ejeta) [reply server/routes/uploadProfileImage.js]: the URL is now sent as res.json to the front end

**Comment by rajivtitus** [comment]
Hey @ibsa-ejeta , can we please get this branch merged when possible. Thanks!

**Comment by ibsa-ejeta** [comment]
merged under branch name "Setup Amazon S3v2"

**Pull request closed without merging**